ci/wheel_smoke_test_ucxx.py

```diff
@@ -0,0 +1,116 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES.
```

Comment:
```suggestion
# SPDX-FileCopyrightText: Copyright (c) 2022-2024, NVIDIA CORPORATION & AFFILIATES.
```

Reply:
Fixed in https://github.com/rapidsai/ucxx/pull/167/commits/3d43a5caef83f5c345bb62910759268c500b38a3